### ISSUE TITLE
Title color

### ISSUE THREAD

**Issue by Himitsu1990** [b-morgan/Skillet-Classic#135]
Hi! Anyway to fix this bright green toxic head color to old good dark green color, as it was before 3.4.1 Ulduar patch? Just like on any screenshots (https://www.curseforge.com/wow/addons/skillet-classic/screenshots). Thank you!
![color](https://user-images.githubusercontent.com/68761515/213459504-f0237d58-1a7f-4e18-b3f7-6e9047f24b1a.png)

**Comment by b-morgan** [comment]
You can thank Blizzard for removing `SetGradientAlpha` from the UI. There were enough other issues that I just removed them rather than research a replacement. Since you have taken the trouble to complain 😀, I have done the research and will return the old look in the next release.

In the mean time, you can pick a color you like by changing line 255 in `...\Addons\Skillet-Classic\UI\MainFrame.lua`:
`	local r,g,b = 0, 0.7, 0; -- dark green`

If you are really ambitious, the new "old" look is
```
	titlebar2:SetHeight(13)
--	titlebar:SetGradientAlpha("VERTICAL", r*0.6,g*0.6,b*0.6,1,r,g,b,1)
	titlebar:SetGradient("VERTICAL", CreateColor(r*0.6,g*0.6,b*0.6,1), CreateColor(r,g,b,1))
	titlebar:SetColorTexture(r,g,b,1)
--	titlebar2:SetGradientAlpha("VERTICAL",r*0.9,g*0.9,b*0.9,1,r*0.6,g*0.6,b*0.6,1)
	titlebar2:SetGradient("VERTICAL", CreateColor(r*0.9,g*0.9,b*0.9,1), CreateColor(r*0.6,g*0.6,b*0.6,1))
	titlebar2:SetColorTexture(r,g,b,1)
```

**Comment by b-morgan** [comment]
Fixed in Skillet-Classic-1.62.
